lib/perl/Genome/WorkflowBuilder/DAG.pm

```diff
@@ -98,21 +112,36 @@ sub _execute_with_workflow {
 }
 
 sub _execute_with_ptero {
-    die "This is not implemented, sorry";
+    my ($self, $inputs, $polling_interval) = @_;
+
+    my $wf_builder = $self->get_ptero_builder($self->name);
+
+    my $wf_proxy = $wf_builder->submit( inputs => $inputs );
+    $wf_proxy->wait(polling_interval => $polling_interval);
```

Comment:
There is an info-level log message in Ptero::HTTP that logs the address each time the polling request is sent, but a log message here wouldn't hurt, do you mean just when we start waiting?